### ISSUE TITLE
Checker does not parse multi-participant tiers

### ISSUE THREAD

**Issue by itang1** [aclew/AAS-minCHAT-Checker#13]
If a tier hosts segments from 2+ indistinguishable speakers, coders are supposed to indicate that by naming the participant as "MULTI" ([see here for the tutorial](https://docs.google.com/presentation/d/14SKqFyiTML9aaPjoKqN6MtHUiJ175xyNR89e31v7Qf4/edit#slide=id.g1f7015eddc_0_41)).

However, the minCHAT checker does not seem to look for errors, capitalizations, or hyphens on tiers where the participant name does not exactly match the tier name.

**Comment by marisacasillas** [comment]
You're right! It'd be nice to add that. I thought about it before, but I got all _sorts_ of strings, e.g., "multi", "MULTI", "multiple speakers", "multi-speaker", etc., so I gave up.

For now, please consider this a shortcoming of the checker where human eyes will have to step in! I will try to find time to fix it in a few weeks. Thank you for posting the issue!
